toolbox/util/common_test.go

```diff
@@ -6,22 +6,22 @@ import (
 )
 
 func TestGetSha256(t *testing.T) {
```

Comment:
Cosmetic, but missing one more rename here 256->512

Reply:
fixed @dougm. updated the anago one too!